src/static/css/index.css

```diff
@@ -1,20 +1,26 @@
-header {
-  /* background-image: url(/static/images/intro-background-fit.svg); */
-  background-image: url(data:image/svg+xml;base64,PHN2ZyB4bWxucz0iaHR0cDovL3d3dy53My5vcmcvMjAwMC9zdmciIHdpZHRoPSIxOTIwIiBoZWlnaHQ9IjEyNTAiIHZpZXdCb3g9IjAgMCAxOTIwIDEyNTAiIHByZXNlcnZlQXNwZWN0UmF0aW89Im5vbmUiPjxkZWZzPjxzdHlsZT4uY2xzLTF7ZmlsbDojYThjYWJhO2ZpbGwtcnVsZTpldmVub2RkO29wYWNpdHk6LjA1fTwvc3R5bGU+PC9kZWZzPjxwYXRoIGNsYXNzPSJjbHMtMSIgZD0iTS0yMzEgODloMjA4OHY2NTFsLTIwODggMzYwVjg5eiIvPjxwYXRoIGNsYXNzPSJjbHMtMSIgZD0iTS00NzAgMjM5aDIwODh2NjUxbC0yMDg4IDM2MFYyMzl6Ii8+PHBhdGggZD0iTS0xNDQtMTloMjA4OHY2NTFMLTE0NCA5OTJWLTE5eiIgZmlsbD0iIzFhMmI0OSIgb3BhY2l0eT0iLjciIGZpbGwtcnVsZT0iZXZlbm9kZCIvPjxwYXRoIGNsYXNzPSJjbHMtMSIgZD0iTS00NzAtODNoMjA4OHY2NTFMLTQ3MCA5MjhWLTgzeiIvPjxwYXRoIGQ9Ik0tMjMyLTQ4aDIwODh2NjUxTC0yMzIgOTYzVi00OHoiIGZpbGw9IiNmMmYyZjIiIG9wYWNpdHk9Ii4wNSIgZmlsbC1ydWxlPSJldmVub2RkIi8+PHBhdGggZD0iTS0xNDI2LTM3MUg2NjJ2NjUxbC0yMDg4IDM2MFYtMzcxeiIgb3BhY2l0eT0iLjEiIGZpbGw9IiNmMmYyZjIiIGZpbGwtcnVsZT0iZXZlbm9kZCIvPjwvc3ZnPg==);
+header.alt-bg {
+  background: #5C687D;
+  background: transparent linear-gradient(#5C687D 70%, transparent 30%);
+  background: transparent url(/static/images/intro-background-fit.svg) bottom left / 100% 100% no-repeat;
+  background: transparent url(data:image/svg+xml;base64,PHN2ZyB4bWxucz0iaHR0cDovL3d3dy53My5vcmcvMjAwMC9zdmciIHdpZHRoPSIxOTIwIiBoZWlnaHQ9IjEyNTAiIHZpZXdCb3g9IjAgMCAxOTIwIDEyNTAiIHByZXNlcnZlQXNwZWN0UmF0aW89Im5vbmUiIHN0eWxlPSJiYWNrZ3JvdW5kOndoaXRlOyI+CiAgPGRlZnM+CiAgICA8c3R5bGU+CiAgICAgIC5jbHMtMXtmaWxsOiNhOGNhYmE7ZmlsbC1ydWxlOmV2ZW5vZGQ7b3BhY2l0eTouMDV9CiAgICA8L3N0eWxlPgogIDwvZGVmcz4KICA8cGF0aCBjbGFzcz0iY2xzLTEiIGQ9Ik0tMjMxIDg5aDIwODh2NjUxbC0yMDg4IDM2MFY4OXoiLz4KICA8cGF0aCBjbGFzcz0iY2xzLTEiIGQ9Ik0tNDcwIDIzOWgyMDg4djY1MWwtMjA4OCAzNjBWMjM5eiIvPgogIDxwYXRoIGQ9Ik0tMTQ0LTE5aDIwODh2NjUxTC0xNDQgOTkyVi0xOXoiIGZpbGw9IiMxYTJiNDkiIG9wYWNpdHk9Ii43NSIgZmlsbC1ydWxlPSJldmVub2RkIi8+CiAgPHBhdGggY2xhc3M9ImNscy0xIiBkPSJNLTQ3MC04M2gyMDg4djY1MUwtNDcwIDkyOFYtODN6Ii8+CiAgPHBhdGggZD0iTS0yMzItNDhoMjA4OHY2NTFMLTIzMiA5NjNWLTQ4eiIgZmlsbD0iI2YyZjJmMiIgb3BhY2l0eT0iLjA1IiBmaWxsLXJ1bGU9ImV2ZW5vZGQiLz4KPC9zdmc+Cg==);
   background-position: bottom left;
   background-repeat: no-repeat;
   background-size: 100% 100%;
-  padding-bottom: 400px;
-}
-
-@media (max-width: 600px) {
-  header {
-    background-size: cover;
-  }
+  padding-bottom: 900px;
+  margin-bottom: -900px;
 }
 
```

Comment:
My thinking is more if the SVG fails because it is not support. I'm ot sure if any Assistive technology (AT) outside of the mainsteam browsers do not render SVGs. Or if SVG backgrounds are ignored for any particular accessibilty settings (e.g. high contrast). See next comment too.